### PR TITLE
chore(size-limit): Add size-limit

### LINKED ISSUE #495: Add size-limit locally and on CI
We should use https://github.com/ai/size-limit in order to keep the bundle size at minimum and ensure it doesn't increase with time.

### PR DESCRIPTION
**Pull request by SaraVieira**
closes #495

**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [x] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**

Add Size limit to local and ci builds to ensure we don't surpass the limit of the library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,15 +8,22 @@
...
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "logo": "cd logo && sketchtool export artboards logo.sketch",
-    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo"
+    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo",
+    "size": "size-limit"
   },
+  "size-limit": [
+    {
+      "path": "dist/index.js",
+      "limit": "5.76 KB"
```
Comment (okonet) [inline package.json]: I'm wondering if there is any way this can be reduced?

File: package.json [hunk 2]
```diff
@@ -8,15 +8,22 @@
...
+    "test": "npm run eslint:src && jest --coverage && npm run size",
     "eslint:src": "eslint ./src ./examples ./*.js",
     "commitmsg": "commitlint -e",
     "precommit": "lint-staged",
     "prepublish": "NODE_ENV=production npm run build && jest",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "logo": "cd logo && sketchtool export artboards logo.sketch",
-    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo"
+    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo",
+    "size": "size-limit"
```
Comment (okonet) [inline package.json]: Should we add `size:why` with the `why` mode?

Comment (SaraVieira) [reply package.json]: Good idea!

File: package.json (hunk 1 above)
Comment (SaraVieira) [reply package.json]: Want me to work on trying to get this smaller a little?

Comment (okonet) [reply package.json]: We could try but let's do in a separate PR.

Comment (SaraVieira) [reply package.json]: I'm gonna do the why and merge this then, cool?

Comment (okonet) [reply package.json]: Yes, please!

Comment (SaraVieira) [reply package.json]: @okonet Done ! Just gonna wait for the ci

File: package.json [hunk 3]
```diff
@@ -8,15 +8,23 @@
...
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "logo": "cd logo && sketchtool export artboards logo.sketch",
-    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo"
+    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo",
+    "size": "size-limit",
+    "size:why": "size-limit --why"
   },
+  "size-limit": [
+    {
+      "path": "dist/index.js",
```
Comment (okonet) [inline package.json]: I was playing around with size-limit and it seems it needs to be the source file in order for the lib to work correctly.

**Pull request merged**